src/App.tsx

```diff
@@ -21,14 +38,26 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {hasError && (
+                <h3 style={{ color: 'tomato' }}>
+                  Todos loading error
+                </h3>
+              )}
+              {isLoading && <Loader />}
+              {!isLoading && !hasError && (
+                <TodoList
+                  isLoading={isLoading}
+                />
+              )}
```

Comment:
Why do you need loading is a props, if you show TodoList component ONLY if !isLoading
It means, that isLoading every time is equally false
```suggestion
              {hasError && (
                <h3 style={{ color: 'tomato' }}>
                  Todos loading error
                </h3>
              )}
              
              {isLoading && <Loader />}
              
              {!isLoading && !hasError && (
                <TodoList />
              )}
```